packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -0,0 +1,164 @@
+layout: Guides
+---
+<ApiLifecycle access="ea" />
+
+This guide explains how to include the `acr_values` parameter in your authorization requests to increase the level of end-user assurance.
+
+---
+
+**Learning outcomes**
+
+* Know the purpose of using the `acr_values` parameter.
+* Learn about the different types of `acr_values` that are supported out of the box.
+* Use an `acr_values` parameter in your authorization request.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* An application that you want to configure `acr_values` for. You can [create a new app integration using AIW](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard) or use an existing one.
+
+---
+
+## Overview
+
+Users want seamless access to certain resources, but organizations want to increase the user’s level of assurance before they access anything sensitive. It’s always difficult to strike a balance between implementing stronger security controls and offering a frictionless experience for your users so that they can easily interact with the application. Using the `acr_values` parameter provides easy access to one layer of resources and secure access to another layer of resources.
+
+The `acr_values` parameter refers to authentication context class reference. Each value defines a specific set of assurance level requirements that the protected resource requires from the authentication event associated with the access and ID tokens.
+
+Today an authorization server relies on [authentication policies](/docs/reference/api/policy/#authentication-policy) to authenticate the user. After the user is authenticated, the authorization server evaluates the scopes and the grant types defined for the application, and then issues the tokens. Although this approach works in many situations, there are several circumstances where more is needed. Resource servers (your protected APIs) can require different authentication strengths or elapsed time frames for different use cases. For example, an eCommerce application requires different authentication strengths depending on whether the item being purchased exceeds a certain threshold. Another example is an application that requires a higher level of assurance before making changes to some sensitive information.
+
+Okta has created predefined `acr_values` that are described in the [**Predefined parameter values** section](#predefined-parameter-values). You can include one of these values, based on your use case, in the client authorization request to request a different authentication assurance. The authorization server returns an access token and/or an ID token that contains the `acr` claim. This claim conveys information about the level of assurance that the user verified at the time of authentication. The resource server can then validate these parameters to ensure that the user verified the required level of assurance.
```

Comment:
I think that this is the first time I've seen bold used on part of the link text. I confess it threw me a bit lol. Would this work, too?
```suggestion
Okta has created predefined `acr_values` that are described in the [Predefined parameter values](#predefined-parameter-values) section. You can include one of these values, based on your use case, in the client authorization request to request a different authentication assurance. The authorization server returns an access token and/or an ID token that contains the `acr` claim. This claim conveys information about the level of assurance that the user verified at the time of authentication. The resource server can then validate these parameters to ensure that the user verified the required level of assurance.
```